Formula/o/openshift-cli.rb

```diff
@@ -1,7 +1,7 @@
 class OpenshiftCli < Formula
   desc "OpenShift command-line interface tools"
   homepage "https://www.openshift.com/"
-  url "https://mirror.openshift.com/pub/openshift-v4/clients/ocp/4.14.9/openshift-client-src.tar.gz"
+  url "https://mirror.openshift.com/pub/openshift-v4/clients/ocp/4.14.10/openshift-client-src.tar.gz"
   sha256 "a2dcdbc17ef649e3fb52b0e51943bc60adfdb126c46b18a8649f768a40afe00d"
```

Comment:
Looks like this is happening again.

Reply:
per [this comment](https://github.com/openshift/oc/issues/1587#issuecomment-1779249514), I wonder if we can just bump it to notify about the other component change

> That would also mean that there is no change in oc but new version is released due to some changes in other components. So that this may not be an issue and can be considered valid.

Or maybe just close this PR (as oc does not have any changes)